[Homework #4] - Css by Max Hirning

Max-Hirning opened the pull request:
- created web page as in figma design
- added theme changing
- added changing active button for portfolio buttons
- added ability to see CV by clicking on button "Download CV"

ilya-ischenko commented on homeworks/max-hirning/04-css/styles/adaptive.css:
```diff
@@ -0,0 +1,67 @@
+@media screen and (min-width: 769px) {
+    .portfolio_buttons {
+        padding: 1.5vw;
+    }
+}
```
Why you use vw for padding? In this case padding value depends on width of the screen
Did you consider it?

Max-Hirning replied:
Yeah, I considered it. It allows for buttons in portfolio area to save padding in the right ratio of button, screen width.

ilya-ischenko commented on homeworks/max-hirning/04-css/styles/main/actions.css:
```diff
@@ -0,0 +1,35 @@
+    justify-content: space-between;
+}
+
+.action_button {
+    width: 45%;
+    border: none;
+    height: 13vw;
+    display: flex;
+    cursor: pointer;
+    max-height: 80px;
+    max-width: 290px;
+    align-items: center;
+    border-radius: 10px;
+    justify-content: center;
+}
+
```
Is it ok that height of your button depends on screen width?

Max-Hirning replied:
Yeah, the thing is that when our screen width shrinks width of button also shrinks, so its' height must do the same action, that's why I made it depending of screen width.

ilya-ischenko commented on homeworks/max-hirning/04-css/styles/main/portfolio.css:
```diff
@@ -0,0 +1,87 @@
+    justify-content: space-between;
+}
+
+.portfolio_example {
+    width: 95%;
+    height: 100vw;
+    display: flex;
+    max-width: 362px;
+    max-height: 226px;
+    margin-block: 15px;
+    text-align: center;
+    border-radius: 22px;
+    align-items: center;
+    flex-direction: column;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url("../../assets/images/portfolio/1.png");
+}
```
The same issue with height, your height values depends on screen width

Max-Hirning replied:
Height must shrinks as well as width that's why I use vw, cause' it must depends on screen width

ilya-ischenko reviewed (commented):
1. Add hover animations to all buttons (chanse bg color, size or box shadow). You can use another websites for inspiration
2. https://ibb.co/Rj3XFv6 - it's hover effect for "project card", so yellow background and text should be visible only when you put cursor over corresponding card

Max-Hirning commented:
Answered on questions

Max-Hirning commented:
1. Added hover animations to all buttons(also added animation for changing theme button after it's clicked )
2. fixed my issue with portfolio card hover

ilya-ischenko reviewed (approved):
Well done!
Approved.

P.S.
Actually your approach with using vw to control element's height works good in this particular case, but do not use it in real projects, especial big one, you will definitely face a lot of issues with it because it's not the best practice to do it, as I know, and it can cause a lot of unexpected behaviour, that's why we have media queries to control styles for different screen sizes
